analyticsdataserver/settings/base.py

```diff
@@ -370,7 +379,9 @@
 
 ########## ANALYTICS DATA API CONFIGURATION
 
-ANALYTICS_DATABASE = 'default'
+ANALYTICS_DATABASE = 'reports'
```

Comment:
This change matches the actual database value getting loaded from both configuration and edx-internal. Has no effect but at least makes things consistent.